app/controllers/tag_articles_controller.rb

```diff
@@ -0,0 +1,65 @@
+class TagArticlesController < ApplicationController
```

Comment:
this is me thinking out loud but I wonder if `TaggedArticlesController` wouldn't be a better name for this class

Reply:
Since the function of this new controller is in the same vein as `StoriesController`, I would think submoduling under it would be the best approach. Maybe `Stories::TaggedArticlesController`

Reply:
+1 on `Stories::TaggedArticlesController`

Reply:
+1 on ` Stories::TaggedArticlesController`, this sounds really fit for the controller.